sn_client/src/file_apis.rs

```diff
@@ -265,6 +265,9 @@ impl Files {
         let mut ordered_read_futures = FuturesOrdered::new();
         let now = Instant::now();
 
+        // The futures are executed concurrently, but the result is returned in the order in which they were inserted.
```

Comment:
Just a silly nit, but can we move this comment  where we call `ordered_read_futures.next()`?

Also can we store the batch_size as a const?